cves/CVE-2012-2775.yml

```diff
@@ -64,7 +64,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability is an out of array write. There is an array of PARCOR
```

Comment:
Could use some more context and explanation of the details. Your audience wouldn't know "PARCOR". I could use a little more research on how the error checking is done. What are the potential exploits that would come from it?